src/commands/publish/upload_form/script.rs

```diff
@@ -0,0 +1,15 @@
+use super::file::{File, ToFile};
+
+#[derive(Debug)]
+pub struct Script {
```

Comment:
likely a struct for this is overkill, i got carried away. nice to have, but `ProjectAssets.files()` could just as easily construct a File with just a `script_path` for now.